src/datajudge/requirements.py

```diff
@@ -1500,12 +1597,25 @@ def add_uniques_subset_constraint(
         Null values in the column are ignored. To assert the non-existence of them use
         the ``NullAbsence`` constraint via the ``add_null_absence_constraint`` helper method
         for ``WithinRequirement``.
+        By default, the null filtering does not trigger if multiple columns are fetched at once.
+        It can be configured in more detail by supplying a custom ``filter_func`` function.
+        Some exemplary implementations are available in this module as ``datajudge.utils.filternull_element``,
```

Comment:
```suggestion
        Some exemplary implementations are available in this module as :func:`~datajudge.utils.filternull_element`,
```

Reply:
Same question as before - is it really 'this' module?

Reply:
Corrected.